lib/vue/src/components/permissions/CPermissionsModal.vue

```diff
@@ -283,6 +285,8 @@ export default {
       // Current role object
       currentRole: undefined,
 
+      roleID: '',
```

Comment:
Instead of defining a new variable, you can rename currentRole to currentRoleID. Since you only use the ID anyway